toolkit/scripts/toolchain/cross/busybox_cross.sh

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Docs used to create this script:
+# http://clfs.org/view/clfs-embedded/x86/final-system/busybox.html
+
+installDir="/opt/cross"
+busyboxInstallDir="/opt/cross/aarch64-mariner-linux-gnu/sysroot"
+buildDir="$HOME/cross"
+scriptDir="$( cd "$( dirname "$0" )" && pwd )"
+
+# Update the username based on your environment
+# sudo chown username ${busyboxInstallDir}
+
+export PATH="${installDir}/bin":$PATH
+
+cd ${buildDir}
+
+# Download source tarballs
+wget https://www.busybox.net/downloads/busybox-1.32.0.tar.bz2
```

Comment:
We should use verified sources. In our case that's usually a source downloaded from a blob storage, for which we already have a signature.